### PR TITLE
fix: getAsync can not init async property

### PR DESCRIPTION
**Pull request by JerrysShan**

### CODE REVIEW

File: src/container.ts [hunk 1]
```diff
@@ -134,6 +134,11 @@ export default class Container implements ContainerType {
         return clazzes.map(clazz => this.get(clazz));
     }
 
+    public getByTagAsync(tag: string) {
+        const clazzes = this.getInjectableByTag(tag);
+        return clazzes.map(clazz => this.getAsync(clazz));
```
Comment (noahziheng) [inline src/container.ts:139]: 应有 Promise.all

File: src/container.ts [hunk 2]
```diff
@@ -55,7 +55,7 @@ export default class Container implements ContainerType {
         if (!md) {
             throw new NotFoundError(id);
         }
-        const instance = this.getValue(md);
+        const instance = this.getValueAsync(md);
```
Comment (noahziheng) [inline src/container.ts:58]: 少 await

**Comment by hyj1991** [comment]
补个测试？

**Review by noahziheng — APPROVED** [review]
LGTM

**Review by hyj1991 — APPROVED** [review]

**Pull request merged**